common/src/main/resources/assets/wynntils/lang/en_us.json

```
@@ -438,6 +439,12 @@
   "feature.wynntils.itemFavorite.lootChestCloseOverride.description": "After how many ESC presses should the chest be closed? Set to 0 to disable.",
   "feature.wynntils.itemFavorite.lootChestCloseOverride.name": "Loot Chest Close Override",
   "feature.wynntils.itemFavorite.name": "Favorite Items",
+  "feature.wynntils.itemFilter.level.invalidRange": "%s: should be an integer or two integers seperated by a dash",
+  "feature.wynntils.itemFilter.level.maxLessThanMin": "%s: min must be less than max",
```

Can you capitalize all errors?

```suggestion
  "feature.wynntils.itemFilter.level.maxLessThanMin": "%s: Min must be less than max",
```